index.html

```diff
@@ -43,9 +43,176 @@ <h3 class="h3">LET’S CONNECT</h3>
+          <div class="title-container">
+            <h2>Multi-Post
+              Stories</h2>
+            <div class="list">
+
+              <span class="list-item-item1">Canopy</span>
+              <span class="list-item">back end Dev</span>
+              <span class="list-item">2015</span>
+            </div>
+          </div>
+          <div class="text-container">
+            <p>A daily selection of privately
+              personalized reads; no accounts or
+              sign-ups required.</p>
+          </div>
+          <div class="languages-list-container">
+            <ul class="lang-container">
+              <li class="langage-item"><span>JavaScript</span></li>
+              <li class="langage-item"><span>HTML</span></li>
+              <li class="langage-item"><span>CSS</span></li>
+            </ul>
+          </div>
+          <button type="button" class="see-btn">See Project</button>
+          <img src="images/portfolio1.jpg" alt="Snapshoot">
+          <div class="title-container">
+            <h2>Tonic</h2>
+            <div class="list">
+
+              <span class="list-item-item1">Canopy</span>
+              <span class="list-item">back end Dev</span>
+              <span class="list-item">2015</span>
+            </div>
+          </div>
+          <div class="text-container">
+            <p>A daily selection of privately
+              personalized reads; no accounts or
+              sign-ups required.</p>
+          </div>
+          <div class="languages-list-container">
+            <ul class="lang-container">
+              <li class="langage-item"><span>JavaScript</span></li>
+              <li class="langage-item"><span>HTML</span></li>
+              <li class="langage-item"><span>CSS</span></li>
+            </ul>
+          </div>
+          <button type="button" class="see-btn">See Project</button>
+          <img src="images/SnapshootPortfolio.png" alt="Snapshoot">
+          <div class="title-container">
+            <h2>Multi-Post
+              Stories</h2>
+            <div class="list">
+
+              <span class="list-item-item1">Canopy</span>
+              <span class="list-item">back end Dev</span>
+              <span class="list-item">2015</span>
+            </div>
+          </div>
+          <div class="text-container">
+            <p>A daily selection of privately
+              personalized reads; no accounts or
+              sign-ups required.</p>
+          </div>
+          <div class="languages-list-container">
+            <ul class="lang-container">
+              <li class="langage-item"><span>JavaScript</span></li>
+              <li class="langage-item"><span>HTML</span></li>
+              <li class="langage-item"><span>CSS</span></li>
+            </ul>
+          </div>
+          <button type="button" class="see-btn">See Project</button>
+        </li>
       </ul>
     </div>
   </section>
+  <section class="about-me">
```

Comment:
About me section should have white background. And the space between My works section and About me section should look like this:

<img width="282" alt="image" src="https://user-images.githubusercontent.com/87897753/159586673-d89b19a0-489e-4660-80a2-c9fbe3a4b56a.png">